dev-docs/bidders/performax.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: bidder
+title: performax
+description Prebid Performax bidder Adapter
+biddercode: performax
+tcfeu_supported: true
+gvl_id: 732
+usp_supported: true
+coppa_supported: false
+schain_supported: true
+userId: all
+media_types: banner
+safeframes_ok: true
+deals_supported: false
+floors_supported: true
+fpd_supported: false
+pbjs: true
+pbs: true
```

Comment:
I don't see a PR open for performax in the Prebid Server repo - are you sure about this?

Reply:
Sorry, you right